src/wp-includes/block-editor.php

```diff
@@ -361,6 +361,92 @@ function _wp_get_iframed_editor_assets() {
 	);
 }
 
+/**
+ * Finds the first occurrence of a specific block in an array of blocks.
+ *
+ * @since 6.3.0
+ *
+ * @param array  $blocks     Array of blocks.
+ * @param string $block_name Name of the block to find.
+ * @return array Found block, or empty array if none found.
+ */
+function wp_get_first_block( $blocks, $block_name ) {
+	foreach ( $blocks as $block ) {
+		if ( $block_name === $block['blockName'] ) {
+			return $block;
+		}
+		if ( ! empty( $block['innerBlocks'] ) ) {
+			$found_block = wp_get_first_block( $block['innerBlocks'], $block_name );
+
+			if ( ! empty( $found_block ) ) {
+				return $found_block;
+			}
+		}
+	}
+
+	return array();
+}
+
+/**
+ * Retrieves Post Content block attributes from the current post template.
+ *
+ * @since 6.3.0
+ * @access private
+ *
+ * @global int $post_ID
+ *
+ * @return array Post Content block attributes or empty array if they don't exist.
+ */
+function wp_get_post_content_block_attributes() {
+	$is_block_theme = wp_is_block_theme();
+
+	global $post_ID;
```

Comment:
See https://github.com/WordPress/wordpress-develop/pull/4614/files#r1232517228, could you move this to the top please?
```suggestion
	global $post_ID;

	$is_block_theme = wp_is_block_theme();
```

Reply:
Oh whoops, missed that! Fixed.